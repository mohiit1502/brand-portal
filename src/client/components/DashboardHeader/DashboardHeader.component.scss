.c-DashboardHeader {
    h3 {
      color: #0B1E40;
      font-weight: bold;
      font-size: 29px;
    }
}
