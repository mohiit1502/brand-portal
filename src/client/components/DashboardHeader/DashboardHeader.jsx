import React from "react";
import PropTypes from "prop-types";
import "./DashboardHeader.component.scss";

const DashboardHeader = props => {
  return (
    <div className="c-DashboardHeader row p-4 h-10 mx-0">
      <div className="col">
        <h3>Hello {props.userInfo.firstName}!</h3>
      </div>
    </div>
  );
};

DashboardHeader.propTypes = {
  userInfo: PropTypes.object
};

export default DashboardHeader;
