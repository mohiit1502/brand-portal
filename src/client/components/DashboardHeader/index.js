import DashboardHeader from './DashboardHeader.jsx';

export default DashboardHeader;