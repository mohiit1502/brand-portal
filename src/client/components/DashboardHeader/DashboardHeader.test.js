/* eslint-disable filenames/match-regex, no-unused-vars, no-undef */
import React from "react";
import DashboardHeader from "./DashboardHeader";

describe("DashboardHeader", () => {
   it("renders without error", () => {

   });
});
