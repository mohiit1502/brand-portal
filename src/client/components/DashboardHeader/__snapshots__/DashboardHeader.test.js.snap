// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardHeader renders without error 1`] = `
<DashboardHeader
  userInfo={Object {}}
>
  <div
    className="c-DashboardHeader row p-4 h-10 mx-0"
  >
    <div
      className="col"
    >
      <h3>
        My Dashboard
      </h3>
    </div>
  </div>
</DashboardHeader>
`;
